lib/capybara_select2/helpers.rb

```diff
@@ -22,7 +22,7 @@ def select2(*args)
       container = if container['class'] =~ /select2-container/
         container
       else
-        container.find('.select2-container')
+        container.find('.select2-container') rescue container.sibling('.select2-container')
```

Comment:
@Hirurg103 Not sure if you agree on the `rescue` but I was not able to find way how to check:

- if it has `.select2-cotainer` child element then use it
- otherwise look at the sibling (this is the case that the container is actually a `<select>` tag and the select2 is initialized as a `<span class="select2-contianer">` next to this element

`find` will raise an exception if the element is not found therefore fallback to `sibling` method.